app/build.gradle

```diff
@@ -63,6 +63,7 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation 'androidx.recyclerview:recyclerview:1.3.0'
+    implementation 'androidx.lifecycle:lifecycle-runtime-testing:2.4.1'
```

Comment:
It seems that there exists a new version of this import. I suggest you to use the last one :

implementation 'androidx.lifecycle:lifecycle-runtime-testing:2.6.1'

Reply:
👍